### PR TITLE
mypy: fix some plugins

### PR DESCRIPTION
**Pull request by MartinBasti**
# Maintainers will complete the following section

- [ ] Commit messages are descriptive enough
- [ ] Code coverage from testing does not decrease and new code is covered
- [ ] JSON/YAML configuration changes are updated in the relevant schema
- [ ] Changes to metadata also update the documentation for the metadata
- [ ] Pull request has a link to an osbs-docs PR for user documentation updates
- [ ] New feature can be disabled from a configuration file

### CODE REVIEW

**Review by ben-alkov — APPROVED** [review]
LGTM

File: atomic_reactor/plugins/pre_add_labels_in_df.py [hunk 1]
```diff
@@ -259,8 +259,10 @@ def add_release_env_var(self, df_parser):
                 self.log.warning("environment release variable %s could not be set because no "
                                  "release label found", release_env_var)
 
-    def add_labels_to_df(self, build_dir: BuildDir) -> str:
+    def add_labels_to_df(self, build_dir: BuildDir):
```
Comment (chmeliik) [inline atomic_reactor/plugins/pre_add_labels_in_df.py:262]: `-> None`

No annotation implicitly means `-> Any`

File: atomic_reactor/plugins/pre_add_labels_in_df.py [hunk 2]
```diff
@@ -179,7 +179,7 @@ def add_as_an_alias(set_to, set_from):
 
         for old, new in self.aliases.items():
             if old not in all_labels:
-                applied_alias = not_applied.append(old)
+                not_applied.append(old)
```
Comment (chmeliik) [inline atomic_reactor/plugins/pre_add_labels_in_df.py:182]: This also changes the logic, but I'm pretty sure the original logic was a bug and it's more correct now (and it's only for a log message anyway)

File: atomic_reactor/plugins/pre_add_labels_in_df.py (hunk 1 above)
Comment (MartinBasti) [reply atomic_reactor/plugins/pre_add_labels_in_df.py]: TIL, thanks

File: atomic_reactor/plugins/pre_add_labels_in_df.py (hunk 2 above)
Comment (MartinBasti) [reply atomic_reactor/plugins/pre_add_labels_in_df.py:182]: Honestly I'm confused with that logic there, why is there even function that always returns `True` :-|

Comment (chmeliik) [reply atomic_reactor/plugins/pre_add_labels_in_df.py:182]: Right? :smile: 

I was going to refactor that - I'm pretty sure `applied_alias` is useless now and the condition could just be `if alias_labels and not_applied`. But then I got lost and gave up, better to leave this skeleton in the closet :smile:

**Review by chmeliik — APPROVED** [review]
LGTM :+1:

**Comment by MartinBasti** [comment]
no release notes, internal changes to mypy annotations

**Pull request merged**